.storybook/welcome-page/footer/footer.component.js

```diff
@@ -10,7 +15,8 @@ const Footer = () => (
         <Corporate>
           <SageIcon />
           <Legal>
-            { I18n.t('footer.legal') }
+            {"\u00A9"} The Sage Group plc. {new Date().getFullYear()}. Licensed
```

Comment:
I think that we don't need curly brackets herefor this string

Reply:
yes, it's special character (tested)